src/effects/Reverb_libSoX.h

```diff
@@ -268,3 +268,34 @@ static void reverb_delete(reverb_t * p)
    fifo_delete(&p->input_fifo);
 }
 
+static void reverb_clear(reverb_t* p)
+{
+   for (size_t c = 0; c < 2; c++)
+   {
+      auto& chn = p->chan[c];
+
+      chn.one_pole->i1 = 0.0;
+      chn.one_pole->o1 = 0.0;
```

Comment:
There are two one_pole objects in an array.   Both must be cleared.

Reply:
good catch! now done